### ISSUE TITLE
AP Digital Testing Portal 1:1 Mockup

### ISSUE THREAD

**Issue by sresri2** [AP-Students/website#46]
This feature should mimic the AP Digital Testing as much as possible so that people can take practice tests in as close to an environment as the real test as possible. The goal is to match the testing portal to the AP testing portal (Bluebook Digital Testing), so our portal is a 1:1 of the real digital testing portal. Note that there may be very minor differences once completed, depending on legal concerns, logos, etc... 

tldr: The goal is to create our own Digital Testing portal for practice tests to match the AP testing portal (Bluebook Digital Testing). Our portal is a 1:1 of the real digital testing portal (or as close as we are allowed to get it).

**Comment by manningwu07** [comment]
I can probably do this one over the weekend (for the front-end at least; using the same object as the quiz)
